FirebaseABTesting.podspec

```diff
@@ -45,10 +45,10 @@ Firebase Cloud Messaging and Firebase Remote Config in your app.
     'HEADER_SEARCH_PATHS' => '"${PODS_TARGET_SRCROOT}"'
   }
   s.dependency 'FirebaseCore', '~> 6.8'
```

Comment:
Delete `GPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1 ` above.

Reply:
Done in dade3e0.